### PR TITLE
2019 updates

### PR DESCRIPTION
**Pull request by norealroots**
- Updates the the README for the workshop
- Added the Apache License 2.0 to the workshop content
- Gave proper attribution the the Al Sweigart, the author of the book the workshop content is based on.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+```
+
+# Invent Your Own Computer Games with Python 3rd Edition
+
+This work is based on the book “Invent Your Own Computer Games with Python 3rd Edition” by Al Sweigart (ISBN-10 0982106017), and is licensed under Creative Commons Attribution-Noncommercial-Share Alike 3.0 United States License.
```
Comment (tofu-rocketry) [inline LICENSE.md]:
```suggestion
This work is based on the book “Invent Your Own Computer Games with Python 3rd Edition” by Al Sweigart (ISBN-10 0982106017) which is licensed under a Creative Commons Attribution-Noncommercial-Share Alike 3.0 United States License.
```

It was ambiguous as to whether you were referring to this work or the book.

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# WorkExp-CompWorkshop
+
+This work is intended for educational purposes.
+
+Copyright 2019 Science and Technology Facilities Council, U.K. Research and Innovation
+
+```
+   Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (tofu-rocketry) [inline LICENSE.md]: I do not think you can release this under an Apache licence if it's based on a CC BY-NC-SA licenced work:

>### Versions 2.0, 2.5, and 3.0
> 
> Your contributions to adaptations of BY-NC-SA 2.0, 2.5 or 3.0 materials may only be licensed under:
> 
>    - The license used for the original work, or a later version of that BY-NC-SA license.
>    - Ported versions of that BY-NC-SA license, the same or later version as the licensed work.

**Review by tofu-rocketry — CHANGES_REQUESTED** [review]
You need to check the licensing.

File: LICENSE.md (hunk 2 above)
Comment (norealroots) [reply LICENSE.md]: I hadn't realised that, I'll move the License to a CC one.

File: LICENSE.md (hunk 1 above)
Comment (norealroots) [reply LICENSE.md]: Agreed, this is a better structure.

File: LICENSE.md (hunk 2 above)
Comment (tofu-rocketry) [reply LICENSE.md]: I forgot the link for my quote for reference: https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses

Comment (norealroots) [reply LICENSE.md]: Thanks - I had found this to check what needed doing. I have changed the workshop's license to a `BY-NC-SA 4.0` license, which meets the criteria of `The license used for the original work, or a later version of that BY-NC-SA license` for a 3.0 license.

**Review by tofu-rocketry — APPROVED** [review]

**Review by jrha — APPROVED** [review]
LGTM!

**Pull request merged**